universal-login-example/src/relayer/TokenGrantingRelayer.js

```diff
@@ -0,0 +1,33 @@
+import {waitForContractDeploy} from './utils';
+import Identity from 'universal-login-contracts/build/Identity';
+import Token from '../../build/Token';
+import Relayer from 'universal-login-relayer';
+import ethers, {Wallet, utils} from 'ethers';
+
+class TokenGrantingRelayer extends Relayer {
+  constructor(config, provider = '', deployerPrivateKey = '', tokenContractAddress = '') {
+    super(config, provider);
+    this.provider = provider || new ethers.providers.JsonRpcProvider(config.jsonRpcUrl, config.chainSpec);
+    this.deployerPrivateKey = deployerPrivateKey || config.privateKey;
+    this.tokenContractAddress = tokenContractAddress || config.tokenContractAddress;
+    this.deployerWallet = new Wallet(this.deployerPrivateKey, this.provider);
```

Comment:
In super, this.provider is already initialized in exactly the same way: 
`this.provider = provider || new ethers.providers.JsonRpcProvider(config.jsonRpcUrl, config.chainSpec);`


    this.wallet = new ethers.Wallet(config.privateKey, this.provider);

Reply:
✅